samples/stock/demo/live-candlestick/demo.js

```diff
@@ -0,0 +1,1024 @@
+    ],
+    [
+        1317894180000,
+        377.36,
+        377.9,
+        376.52,
+        376.75
+    ],
+    [
+        1317894240000,
+        376.83,
+        377.73,
+        376.71,
+        376.98
+    ],
+    [
+        1317894300000,
+        377,
+        377.69,
+        376.87,
+        377.1212
+    ],
+    [
+        1317894360000,
+        377.225,
+        377.33,
+        376.01,
+        376.26
+    ],
+    [
+        1317894420000,
+        376.42,
+        376.64,
+        375.55,
+        375.5534
+    ],
+    [
+        1317894480000,
+        375.74,
+        375.94,
+        374.77,
+        375.3
+    ],
+    [
+        1317894540000,
+        375.3313,
+        376,
+        374.92,
+        375.06
+    ],
+    [
+        1317894600000,
+        375.11,
+        375.46,
+        374.82,
+        374.92
+    ],
+    [
+        1317894660000,
+        374.82,
+        375.68,
+        374.64,
+        375.668
+    ],
+    [
+        1317894720000,
+        375.62,
+        376.13,
+        375.46,
+        376.13
+    ],
+    [
+        1317894780000,
+        376.14,
+        376.6,
+        375.89,
+        376.34
+    ],
+    [
+        1317894840000,
+        376.39,
+        376.39,
+        375.55,
+        375.99
+    ],
+    [
+        1317894900000,
+        376,
+        376.28,
+        375.42,
+        376.21
+    ],
+    [
+        1317894960000,
+        376,
+        377.38,
+        375.7,
+        376.591
+    ],
+    [
+        1317895020000,
+        376.59,
+        377.46,
+        376.57,
+        376.9348
+    ],
+    [
+        1317895080000,
+        376.9481,
+        377.749,
+        376.84,
+        377.563
+    ],
+    [
+        1317895140000,
+        377.452,
+        377.65,
+        376.43,
+        376.78
+    ],
+    [
+        1317895200000,
+        376.94,
+        377.01,
+        375.75,
+        375.98
+    ],
+    [
+        1317895260000,
+        376.27,
+        377.29,
+        375.95,
+        376.98
+    ],
+    [
+        1317895320000,
+        376.9962,
+        377.3,
+        376.69,
+        376.71
+    ],
+    [
+        1317895380000,
+        376.75,
+        377.5,
+        376.75,
+        377.41
+    ],
+    [
+        1317895440000,
+        377.26,
+        377.49,
+        376.89,
+        377.368
+    ],
+    [
+        1317895500000,
+        377.345,
+        378,
+        377.17,
+        378
+    ],
+    [
+        1317895560000,
+        377.97,
+        378.3199,
+        377.68,
+        377.97
+    ],
+    [
+        1317895620000,
+        378.01,
+        378.07,
+        377.25,
+        377.37
+    ],
+    [
+        1317895680000,
+        377.37,
+        377.75,
+        377.05,
+        377.12
+    ],
+    [
+        1317895740000,
+        377.16,
+        377.79,
+        377.01,
+        377.4512
+    ]
+];
+
+// Imitate getting point from backend:
+function getNewPoint(i) {
+    const lastPoint = data[data.length - 1];
```

Comment:
Great job!  If we use this function on an empty dataset it will break, so maybe there should be something to handle that case or prevent any errors.

Reply:
Thank you! I agree with your point here, that it won't work without the data - but since this function is created to mimic getting point from the backend, I don't think we should include any no-data prevention here. In real world scenario the implementation would be drastically different since we would work on real values, not values based on previous points.

With such a `if(data)` check, we couldn't even create a new point, because then it would have to have random values (since now they are based on previous point).

But I'll leave the final decision to @pawelfus 😅

Reply:
No strong opinion here, you are both right. Since it's a demo, then I would say less code is better